src/init/features/dataconnect/resolver.ts

```diff
@@ -0,0 +1,144 @@
+import * as clc from "colorette";
+import * as fs from "fs-extra";
+import { join, relative } from "path";
+import * as yaml from "yaml";
+
+import { input, select } from "../../../prompt";
+import { Setup } from "../..";
+import { logBullet, newUniqueId } from "../../../utils";
+import { Config } from "../../../config";
+import { loadAll } from "../../../dataconnect/load";
+import { DataConnectYaml, SchemaYaml, ServiceInfo } from "../../../dataconnect/types";
+import { parseServiceName } from "../../../dataconnect/names";
+import * as experiments from "../../../experiments";
+import { isBillingEnabled } from "../../../gcp/cloudbilling";
+import { trackGA4 } from "../../../track";
+import { Source } from ".";
+
+export interface ResolverRequiredInfo {
+  id: string;
+  uri: string;
+  serviceInfo: ServiceInfo;
+}
+
+export async function askQuestions(setup: Setup, config: Config): Promise<void> {
+  if (!experiments.isEnabled("fdcwebhooks")) {
```

Comment:
Shall we hide this option if the experiment is off in `src/commands/init.ts`?

To be cautious, I don't remember if that list is printed out in other spot.